### DISCUSSION TITLE
Debugbar env=dev not show if request == 200

### DISCUSSION THREAD

**Discussion by ghost** [symfony/symfony#42922] [Q&A] [answered]
Hi,

The title says it all, the debugbar is displayed if I type a bogus link, but if the request is ok, the bar is no longer displayed. Am I the only one ? Is there an option to add to force its display?

Thanking you in advance for your insight and looking forward to reading you,

Regards,

**Comment by ahundiak** [comment, marked as answer]
First thing to check is to see if your request is generating valid html.  You can examine this in the browser.  Typically an html page is sent along with some debug javascript.  The javascript in turn executes some queries and then displays the bar.  So if something prevents the javascript from executing then no bar.

It's a bit puzzling when you say the bar is displayed on a bogus link.  I would have expected a big route not found page to show up unless you are matching all possible routes.

**Reply by ghost** [reply]
> First thing to check is to see if your request is generating valid html. You can examine this in the browser. Typically an html page is sent along with some debug javascript. The javascript in turn executes some queries and then displays the bar. So if something prevents the javascript from executing then no bar.

Ok ... Thank you very much for putting me on the track ... Before, when we started doing a make: controller, it created the template with ```{% extends('base.html.twig') %}```, now no ... I hadn't paid attention to these details. It was when you mentioned loading JS that I understood what had happened. Thanks again ;)
